### PR TITLE
Feat: implement new command EXZPOPMAX and EXZPOPMIN.

### PR DESCRIPTION
**Pull request by RinChanNOWWW**
Signed-off-by: RinChanNOWWW <hzy427@gmail.com>

relate issue: #4

### CODE REVIEW

File: src/tairzset.c [hunk 1]
```diff
@@ -1635,6 +1638,96 @@ void exZunionInterDiffGenericCommand(RedisModuleCtx *ctx, RedisModuleString **ar
...
+    long rangelen = (count > llen) ? llen : count;
+    RedisModule_ReplyWithArray(ctx, rangelen * 2);
+
+    /* Remove the element. */
+    do {
+        m_zskiplist *zsl = tair_zset_obj->zsl;
+        m_zskiplistNode *zln;
+
+        /* Get the first or last element in the sorted set. */
+        zln = (where == POP_MAX ? zsl->tail :
```
Comment (chenyang8094) [inline src/tairzset.c]: format

File: src/tairzset.c [hunk 2]
```diff
@@ -1635,6 +1638,96 @@ void exZunionInterDiffGenericCommand(RedisModuleCtx *ctx, RedisModuleString **ar
...
+    }
+
+    RedisModuleKey *key = RedisModule_OpenKey(ctx, argv[1], REDISMODULE_WRITE);
+    int type = RedisModule_KeyType(key);
+    if (REDISMODULE_KEYTYPE_EMPTY != type && RedisModule_ModuleTypeGetType(key) != TairZsetType) {
+        RedisModule_ReplyWithError(ctx, REDISMODULE_ERRORMSG_WRONGTYPE);
+        return REDISMODULE_OK;
+    }
+
+    if (type == REDISMODULE_KEYTYPE_EMPTY || key == NULL) {
```
Comment (chenyang8094) [inline src/tairzset.c]: I think `key == NULL` is no need.

**Review by chenyang8094 — CHANGES_REQUESTED** [review]
LGTM

File: src/tairzset.c (hunk 2 above)
Comment (RinChanNOWWW) [reply src/tairzset.c]: I see.

**Review by chenyang8094 — APPROVED** [review]

**Pull request merged**